packages/headless/src/features/configuration/configuration-slice.test.ts

```diff
@@ -1,4 +1,7 @@
-import {restoreTab} from '../search-parameters/search-parameter-actions.js';
+import {
+  restoreSearchParameters,
+  restoreTab,
```

Comment:
Is the restoreTab action still relevant at all following this change? I think not.

It's still handled in the configuration and tab-set slices, but it's not getting dispatched through any headless controller anymore.

We should mark the action as deprecated in search-parameters-actions-loader.ts and create a v4 Jira to remove it.